github/resource_github_organization_ruleset.go

```diff
@@ -23,7 +23,8 @@ func resourceGithubOrganizationRuleset() *schema.Resource {
 			State: resourceGithubOrganizationRulesetImport,
 		},
 
-		SchemaVersion: 1,
+		SchemaVersion: 2,
```

Comment:
I'm not sure why we need to do this? If we don't then I'd say that this goes against the principle of least surprise.